### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by Pacane**
I have refactored the database_test to use async/await. Now locally, I only have 1 test failing but travis seems to have more failing. I still don't know why. I'll try to find out why.

@vadimtsushko would it be possible to have admin access to travis?

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ authors:
 - Ted Sander <ted@tedsander.com>
 - Paul Evans <pevans@sonardesign.com>
 - Stéphane Este-Gracias <sestegra@gmail.com>
+- Joel Trottier-Hebert <iamquebecois@gmail.com>
```
Comment (Pacane) [inline pubspec.yaml]: @vadimtsushko hope you don't mind :)

**Comment by Pacane** [comment]
@vadimtsushko Sadly you'll probably have to check it out to review it, seems like it's too large. I could probably make multiple PRs, would you prefer that?

**Comment by Pacane** [comment]
I believe Travis is the problem, I've made a Codeship instance and this build seems to pass. https://codeship.com/projects/94593/builds/9307030

I'll wait until you give me access to travis to check what commands it runs and then try to reproduce it.

**Comment by Scorpiion** [comment]
@Pacane just so you know, I can't see your Codeship output. I think only your account can see it.

**Pull request merged**